src/main/java/io/oneko/kubernetes/impl/DeploymentManagerImpl.java

```diff
@@ -78,7 +78,7 @@ public ReadableProjectVersion deploy(final WritableProjectVersion version) {
 				}
 
 				final List<InstallStatus> installStatuses = HelmCommandUtils.install(version);
-				log.info("Installing helm releases {} for {}",
+				log.info("installing helm releases {} for {}",
```

Comment:
Didn't see this in the first review but parameters should be at the end of the message, same applies for the other log message.